src/python/test/test_wasserstein_distance.py

```diff
@@ -8,41 +8,66 @@
       - YYYY/MM Author: Description of the modification
 """
 
-from gudhi.wasserstein import wasserstein_distance
+from gudhi.wasserstein import wasserstein_distance as pot
+from gudhi.hera import wasserstein_distance as hera
 import numpy as np
 
 __author__ = "Theo Lacombe"
 __copyright__ = "Copyright (C) 2019 Inria"
 __license__ = "MIT"
 
-
-def test_basic_wasserstein():
+def _basic_wasserstein(wasserstein_distance, delta, test_infinity=True):
     diag1 = np.array([[2.7, 3.7], [9.6, 14.0], [34.2, 34.974]])
     diag2 = np.array([[2.8, 4.45], [9.5, 14.1]])
     diag3 = np.array([[0, 2], [4, 6]])
     diag4 = np.array([[0, 3], [4, 8]])
-    emptydiag = np.array([[]])
+    emptydiag = np.array([])
+
+    # We just need to handle positive numbers here
+    def approx(a, b):
+        f = 1 + delta
+        return a <= b*f and b <= a*f
```

Comment:
You could also use [pytest approx](https://docs.pytest.org/en/latest/reference.html#pytest-approx)

Reply:
Done.